fg2: <main> function()
  block 0: [entry]
    node 0: constant[undefined,v1] (test/flowgraphbuilder/flowgraph_builder0201e.js)
    ->[block 1]
  block 1:
    node 1: read-variable['o',v2,-] (test/flowgraphbuilder/flowgraph_builder0201e.js:1:11)
    ->[block 2]
  block 2:
    node 2: begin-for-in[v2,v3] (test/flowgraphbuilder/flowgraph_builder0201e.js:1:1)
    ->[block 3]
  block 3:
    node 3: has-next[v3,v4] (test/flowgraphbuilder/flowgraph_builder0201e.js:1:1)
    node 4: if[v4](true-block:4,false-block:7) (test/flowgraphbuilder/flowgraph_builder0201e.js:1:1)
    ->[block 4,block 7] ~>[block 6]
  block 4:
    node 5: next-property[v3,v5] (test/flowgraphbuilder/flowgraph_builder0201e.js:1:6)
    node 6: write-variable[v5,'p']* (test/flowgraphbuilder/flowgraph_builder0201e.js:1:6)
    ->[block 5] ~>[block 6]
  block 5:
    node 7: end-for-in(begin-node:2)* (test/flowgraphbuilder/flowgraph_builder0201e.js:3:1)
    ->[block 2]
  block 6:
    node 8: end-for-in(begin-node:2)* (test/flowgraphbuilder/flowgraph_builder0201e.js:3:1)
    ->[block 9]
  block 7:
    node 9: end-for-in(begin-node:2)* (test/flowgraphbuilder/flowgraph_builder0201e.js:3:1)
    ->[block 8]
  block 8: [exit-ordinary]
    node 10: return[v1] (test/flowgraphbuilder/flowgraph_builder0201e.js)
    ->[]
  block 9: [exit-exceptional]
    node 11: exceptional-return (test/flowgraphbuilder/flowgraph_builder0201e.js)
    ->[]

test/flowgraphbuilder/flowgraph_builder0201e.js:1:11: [definite] ReferenceError, reading absent variable o
test/flowgraphbuilder/flowgraph_builder0201e.js:1:1: [definite] Unreachable code
test/flowgraphbuilder/flowgraph_builder0201e.js: [maybe] Uncaught exception, constructed at [test/flowgraphbuilder/flowgraph_builder0201e.js:1:11]
